enterprise/frontend/src/embedding-sdk/cli/steps/ask-if-has-database.ts

```diff
@@ -0,0 +1,23 @@
+import toggle from "inquirer-toggle";
```

Comment:
I thought the plan was to just fallback to using the sample db, and skipping this question, did I mis-read the [thread](https://metaboat.slack.com/archives/C063Q3F1HPF/p1730894065399059?thread_ts=1730304237.098069&cid=C063Q3F1HPF)?